js/index.js

```diff
@@ -120,6 +120,15 @@ function renderStats() {
   });
 }
 
+function renderEmoji(code) {
+  var template = document.getElementById('emoji-template').innerHTML;
```

Comment:
Getting the element over and over again might be an overhead. Just cache it like the others [here](https://github.com/kunyan/bower.github.io/blob/fa02f809c894269bb63abc1a2d2cca492fcfebd8/js/index.js#L145).